CONTRIBUTE.md

```diff
@@ -0,0 +1,30 @@
+# How to Contribute to AlwaysOn
+
+## Content Changes and Pull Requests
+
+To add or edit content within the AlwaysOn repositories, please take a fork of a repository to iterate on changes before subsequently opening a Pull Request (PR) to get your forked branch merged into the main branch for that AlwaysOn repository. Your PR will be reviewed by core engineers working on the AlwaysOn project, and once approved, your content accessible to everybody.
+
+> **Important!** Please make sure that your PR is focused on a specific area of AlwaysOn to facilitate a targeted review, as this will speed up the process to get your changes merged into our repository.
+
+## Content Structure
+
+The AlwaysOn project is separated into **3** different repositories:
+
+* [AlwaysOn](./README.md): contains the AlwaysOn design methodology, covering the design pattern ad approach to guide readers to defining a target AlwaysOn architecture.
```

Comment:
lets switch from * to - to stay consistent

Reply:
done